request/vectorDelete.go

```diff
@@ -7,12 +7,12 @@ import (
 )
 
 type VectorDelete struct {
-	IndexName string            `json:"-"`
-	ProjectID string            `json:"-"`
-	ID        *string           `json:"id,omitempty"`
-	DeleteAll *bool             `json:"deleteAll,omitempty"`
-	Namespace *string           `json:"namespace,omitempty"`
-	Filter    map[string]string `json:"filter,omitempty"`
+	IndexName string         `json:"-"`
+	ProjectID string         `json:"-"`
+	ID        *string        `json:"id,omitempty"`
+	DeleteAll *bool          `json:"deleteAll,omitempty"`
+	Namespace *string        `json:"namespace,omitempty"`
+	Filter    map[string]any `json:"filter,omitempty"`
```

Comment:
Could you please define a custom type for Filter that aliases `map[string]any`and use everywhere it is needed?

Reply:
@JanRuettinger thanks for your contribution!

Reply:
> Could you please define a custom type for Filter that aliases `map[string]any`and use everywhere it is needed?

Good idea! Will make the changes later today.

Reply:
@JanRuettinger is there any news?

Reply:
I wasn't quite sure where to add the type alias. I have added it in the request.go file. What do you think?